pkg/util/util.go

```diff
@@ -144,8 +147,20 @@ func SHA256(r io.Reader) (string, error) {
 
 // CurrentPlatform returns the v1.Platform on which the code runs
 func currentPlatform() v1.Platform {
+	var archVariant string
+	if strings.HasPrefix(runtime.GOARCH, "arm") {
```

Comment:
I am little confused how this works. 
IIUC, we build a Kaniko binary on my local environment. It would contain the `arm` and `archVariant` thats on my machine right?

Does this mean, our Dockerfile should also change?
```
FROM golang:1.12
ARG GOARCH=amd64
```